changelog/fragments/upgrade-operator-framework-api.yaml

```diff
@@ -0,0 +1,92 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Upgrade operator-framework/api version from `v0.3.13` to [`v0.3.20`](https://github.com/operator-framework/api/releases/tag/v0.3.20).
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Upgrade operator-framework/api version from `v0.3.13` to `0.3.20`
+      body: >
+        In the `go.mod` file replace `operator-framework/api v0.3.13`
+        with `operator-framework/api v0.3.20` and then run `go mod tidy`.
+
```

Comment:
This change isn't user-facing.
```suggestion
```

Reply:
Done. A similar change was mentioned for [kb upgrade](https://github.com/operator-framework/operator-sdk/blob/282f2a5cff21d00ed2697ca507a58dcade876aaf/changelog/fragments/upgrade-kb.yaml#L4-L25), so had included it here.

Reply:
Projects don't have a `operator-framework/api` dependency, but they do have a `controller-runtime` dependency, which was why that was added in #4062.

Reply:
Ah.okay... got it..